### ISSUE TITLE
about ASLSingleLabel

### ISSUE THREAD

**Issue by chengerr** [Alibaba-MIIL/ASL#68]
I find that , when gamma+=0, ASLSingleLabel  is actually Equivalent to torch.nn.CrossEntropyLoss  .
And , 
Due to the existence of the code“loss = - self.targets_classes.mul(log_preds)” in  losses.py ,  the value gamma-  does not count, so would you please tell me the difference between ASLSingleLabel(gamma+>0) and Focal loss in your experiment (Table 12 in your paper) ?

**Comment by mrT23** [comment]
1) read more thourughly the code:
        asymmetric_w = torch.pow(1 - xs_pos - xs_neg,
                                 self.gamma_pos * targets + self.gamma_neg * anti_targets)
                                 
2) the article uses: 
_class AsymmetricLoss(nn.Module):_           
(multi-label)
